plugins/modules/postgresql_owner.py

```diff
@@ -384,14 +659,18 @@ def __role_exists(self, role):
 # Module execution.
 #
 
+VALID_OBJ_TYPES = ('aggregate', 'collation', 'conversion', 'database', 'domain', 'event_trigger', 'foreign_data_wrapper',
+                   'foreign_table', 'function', 'language', 'large_object', 'matview', 'procedure', 'publication',
+                   'routine', 'schema', 'sequence', 'server', 'statistics', 'table', 'tablespace', 'text_search_configuration',
+                   'text_search_dictionary', 'type', 'view')
```

Comment:
i think we should put this constant right after the packages-import section

Reply:
Done